File: v1/src/main/java/com/api/v1/payment/PaymentResponseDto.java
package com.api.v1.payment;

import com.api.v1.car.dtos.CarResponseDto;
import com.api.v1.creditcar.dtos.CreditCardResponseDto;
import com.api.v1.customer.dtos.CustomerResponseDto;

public record PaymentResponseDto(
        CustomerResponseDto customer,
        CarResponseDto car,
        CreditCardResponseDto creditCard,
        String paidAt
) {
}

File: v2/src/main/kotlin/com/api/v2/payment/dtos/PaymentResponseDto.kt
package com.api.v2.payment.dtos

import com.api.v2.car.dtos.CarResponseDto
import com.api.v2.creditcard.dtos.CreditCardResponseDto
import com.api.v2.customer.dtos.CustomerResponseDto
import java.math.BigInteger

data class PaymentResponseDto(
    val orderNumber: BigInteger,
    val customer: CustomerResponseDto,
    val car: CarResponseDto,
    val creditCard: CreditCardResponseDto,
    val paidAt: String
)